tests/utils/testutils.py

```diff
@@ -57,6 +57,7 @@
 E2E_TESTS_ROOT = TESTS_ROOT / E2E_TESTS_FOLDER
 UNIT_TESTS_FOLDER = pathlib.Path('unittests')
 UNIT_TESTS_ROOT = TESTS_ROOT / UNIT_TESTS_FOLDER
+EXTENSION_TESTS_FOLDER = pathlib.Path('extension_tests')
```

Comment:
We'll be adding more tests for each supported deferred binding type / future extension types. I thought it might be cleaner to keep them separate instead of making our E2E tests folder really large